### PR TITLE
First SSI pages, TODOs added

### PR DESCRIPTION
**Pull request by kirahsapong**

### CODE REVIEW

File: docs/SSI/introduction.md [hunk 1]
```diff
@@ -0,0 +1,96 @@
...
+hide_title: true
+---
+
+import Link from '@docusaurus/Link';
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<div className="prose prose-pink">
+
+# Self Sovereign Identity Service (SSIS) API
```
[the remark this answers is not in the document]
Comment (angiejones) [reply docs/SSI/introduction.md:13]: agreed

File: docs/SSI/introduction.md [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+- Interacting with the standards around Verifiable Credentials such as: Credential Revocations, Applying for Credentials, Exchanging Credentials, and Data Schemas (for credentials and other verifiable data).
+
+## Run SSI Service
+
+:::info
+The SSI Service is packaged as a Docker container and a Docker Compose file is included to make it simple to run the service locally. First make sure you have [Docker downloaded](https://www.docker.com/products/docker-desktop/) and running on your desktop.
+:::
+
+Locally clone the SSI Service repo:
+```code
```
Comment (angiejones) [inline docs/SSI/introduction.md:53]: use `bash` instead of `code` so that we get syntax coloring

File: docs/SSI/introduction.md [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+git clone https://github.com/TBD54566975/ssi-service.git
+```
+
+Run the Docker Compose file:
+```code
```
Comment (angiejones) [inline docs/SSI/introduction.md:58]: use `bash` instead of `code` so that we get syntax coloring

File: docs/SSI/introduction.md [hunk 4]
```diff
@@ -0,0 +1,96 @@
...
+cd ssi-service/build
+docker-compose up --build
+```
+
+If you'd like to confirm the SSI service and sub-services are functioning, check the health and readiness endpoints:
+
+```code
```
Comment (angiejones) [inline docs/SSI/introduction.md:65]: use `bash` instead of `code` so that we get syntax coloring

File: docs/SSI/introduction.md [hunk 5]
```diff
@@ -0,0 +1,96 @@
...
+curl localhost:8080/health
+```
+
+The following response should be returned:
+
+```code
```
Comment (angiejones) [inline docs/SSI/introduction.md:71]: use `js` instead of `code` so that we get syntax coloring

File: docs/SSI/ssi-console.md [hunk 6]
```diff
@@ -0,0 +1,99 @@
...
+- Create and Collect Decentralized Identifiers (DIDs)
+- Issue and Verify Verifiable Credentials (VCs)
+
+## Installation
+:::info
+The SSI Console leverages the SSI Service which is packaged as a Docker container. Make sure you have [Docker downloaded](https://www.docker.com/products/docker-desktop/) and running on your desktop.
+:::
+
+### 1. Clone and Run SSI Console
+```code
```
Comment (angiejones) [inline docs/SSI/ssi-console.md:15]: use `bash` instead of `code` so that we get syntax coloring

**Review by angiejones — CHANGES_REQUESTED** [review]

File: docs/SSI/introduction.md [hunk 7]
```diff
@@ -0,0 +1,96 @@
...
+## Core Functionality
+- Create and manage [Decentralized Identifiers](https://developer.tbd.website/docs/web5/learn/decentralized-identifiers)
+- Create and manage [Verifiable Credentials](https://www.w3.org/TR/vc-data-model/)
+- Credential Suspension
+- Interacting with the standards around Verifiable Credentials such as: Credential Revocations, Applying for Credentials, Exchanging Credentials, and Data Schemas (for credentials and other verifiable data).
+
+## Run SSI Service
+
+:::info
+The SSI Service is packaged as a Docker container and a Docker Compose file is included to make it simple to run the service locally. First make sure you have [Docker downloaded](https://www.docker.com/products/docker-desktop/) and running on your desktop.
```
[the remark this answers is not in the document]
Comment (bobbilee19) [reply docs/SSI/introduction.md:49]: should i put this in all in the info block though? it'll be big, we'll reassess after I do.

File: docs/SSI/ssi-console.md [hunk 8]
```diff
@@ -0,0 +1,99 @@
...
+git clone https://github.com/TBD54566975/ssi-console.git
+cd ssi-console
+npm install
+npm start
+```
+
+Head to `localhost:3000` and follow the guide below for all actions you can take from the homepage.
```
Comment (angiejones) [inline docs/SSI/ssi-console.md:22]: ~Head to `localhost:3000`~ Open `localhost:3000` in a browser

File: docs/SSI/ssi-console.md [hunk 9]
```diff
@@ -0,0 +1,99 @@
...
+
+## Create a Verifiable Credential (VC):
+### 1. Navigate to `localhost:3000/credentials` and click `Create New`
+
+### 2. Add name and description:
+Feel free to use the default placeholders or describe your new VC.
+
+### 3. Prep VC Schema:
+- Set 'Schema' to 'New Data set'
+- Click 'Try sample input' and fill in the reciever of the VC's first and last name.
```
[the remark this answers is not in the document]
Comment (bobbilee19) [reply docs/SSI/ssi-console.md:53]: yess perfect work change

File: docs/SSI/ssi-console.md [hunk 10]
```diff
@@ -0,0 +1,99 @@
...
+## Core Functionality
+- Create and Collect Decentralized Identifiers (DIDs)
+- Issue and Verify Verifiable Credentials (VCs)
+
+## Installation
+:::info
+The SSI Console leverages the SSI Service which is packaged as a Docker container. Make sure you have [Docker downloaded](https://www.docker.com/products/docker-desktop/) and running on your desktop.
+:::
+
+### 1. Clone and Run SSI Console
```
Comment (angiejones) [inline docs/SSI/ssi-console.md:14]: If this is the only step for install, let's move the number `1.`

File: docs/SSI/ssi-console.md [hunk 11]
```diff
@@ -0,0 +1,99 @@
...
+```
+
+Head to `localhost:3000` and follow the guide below for all actions you can take from the homepage.
+
+## Create a Decentralized Identifier (DID):
+_To learn more about DIDs see [here](https://developer.tbd.website/docs/web5/learn/decentralized-identifiers)._
+
+Upon installation a DID will be created by default, but you can follow the steps below to create one for yourself.
+
+### 1. Navigate to `localhost:3000/dids` and click `Add New`
```
Comment (angiejones) [inline docs/SSI/ssi-console.md:29]: instead of enclosing the url in ticks, hyperlink it to http://localhost:3000/

File: docs/SSI/ssi-console.md [hunk 12]
```diff
@@ -0,0 +1,99 @@
...
+
+### 2. Set DID Method:
+There are three different DID methods to choose from: Ion, Web, and Key. Select 'Key' for now.
+
+_To learn more about these methods click [here](https://www.w3.org/TR/did-spec-registries/#did-methods)._
```
Comment (angiejones) [inline docs/SSI/ssi-console.md:34]: Link to our own description: https://developer.tbd.website/docs/web5/learn/decentralized-identifiers#methods

Also, for SEO purposes try to avoid hyperlinking the word "here". Can rephrase this to:

Learn more about these [DID methods](https://developer.tbd.website/docs/web5/learn/decentralized-identifiers#methods).

File: docs/SSI/ssi-console.md (hunk 9 above)
Comment (angiejones) [inline docs/SSI/ssi-console.md:53]: Some of the click steps use ticks and this one uses quotes. Should choose one format and use it throughout

File: docs/SSI/ssi-console.md [hunk 13]
```diff
@@ -0,0 +1,99 @@
...
+### 1. Navigate to `localhost:3000/credentials` and click `Create New`
+
+### 2. Add name and description:
+Feel free to use the default placeholders or describe your new VC.
+
+### 3. Prep VC Schema:
+- Set 'Schema' to 'New Data set'
+- Click 'Try sample input' and fill in the reciever of the VC's first and last name.
+
+_See [here](https://developer.tbd.website/docs/apis/ssi-service#tag/SchemaAPI) for the fields you can add to this schema._
```
Comment (angiejones) [inline docs/SSI/ssi-console.md:55]: See [fields you can add to this schema](https://developer.tbd.website/docs/apis/ssi-service#tag/SchemaAPI)

File: docs/SSI/ssi-console.md [hunk 14]
```diff
@@ -0,0 +1,99 @@
...
+
+### 4. Set Presentation Definition / Requirements:
+Click 'Try sample input' and leave the default added JSON for now.
+_See [here](https://developer.tbd.website/docs/apis/ssi-service#tag/PresentationDefinitionAPI/paths/~1v1~1presentations~1definitions/put) to learn more about presentation definitions._
```
Comment (angiejones) [inline docs/SSI/ssi-console.md:59]: Learn more about [presentation definitions](https://developer.tbd.website/docs/apis/ssi-service#tag/PresentationDefinitionAPI/paths/~1v1~1presentations~1definitions/put).

File: docs/SSI/ssi-console.md [hunk 15]
```diff
@@ -0,0 +1,99 @@
...
+
+### Issue a VC:
+  - `Subject DID`: Add your employees DID.
+  - `Subject Data`: Fill in your employees first and last name.
+  - Decide whether you want this DID to be `revocable` or `suspendable`
+  - Set an `expiration` date if you want it to expire.
+  - `Issuer DID`: Chose one of the employers' DIDs.
+
+## Verify a Credential:
+### 1. Navigate to: `localhost:3000/verification`
```
Comment (angiejones) [inline docs/SSI/ssi-console.md:85]: - remove number `1.` since it's just one step.
- hyperlink to localhost instead of codifying it

File: docs/SSI/ssi-console.md [hunk 16]
```diff
@@ -0,0 +1,99 @@
...
+### Json:
```
Comment (angiejones) [inline docs/SSI/ssi-console.md:86]: JSON

File: docs/SSI/ssi-console.md [hunk 17]
```diff
@@ -0,0 +1,99 @@
...
+Click `try sample input`
+_See [here](https://developer.tbd.website/docs/apis/ssi-service#tag/PresentationDefinitionAPI/paths/~1v1~1presentations~1definitions/put) for how to configure your own Json._
```
Comment (angiejones) [inline docs/SSI/ssi-console.md:88]: [Configure your own JSON](https://developer.tbd.website/docs/apis/ssi-service#tag/PresentationDefinitionAPI/paths/~1v1~1presentations~1definitions/put).

**Review by angiejones — CHANGES_REQUESTED** [review]

File: docs/SSI/introduction.md
Comment (angiejones) [inline docs/SSI/introduction.md:1]: Can we change the sidebar title from "Introduction" to "Run SSI Service"?

**Review by angiejones — CHANGES_REQUESTED** [review]

File: docs/SSI/introduction.md [hunk 18]
```diff
@@ -0,0 +1,96 @@
...
+        },
+        "schema": {
+            "status": "ready"
+        }
+    }
+}
+```
+
+## Next Steps:
+- Learn how to use the SSI Service to [manually issue Verifiable Credentials](https://developer.tbd.website/blog/issue-verifiable-credential-manually).
```
Comment (angiejones) [inline docs/SSI/introduction.md:94]: If blog post is outdated, let's remove this for now

File: docs/SSI/ssi-console.md (hunk 10 above)
Comment (bobbilee19) [reply docs/SSI/ssi-console.md:23]: great catch

File: docs/SSI/ssi-console.md (hunk 17 above)
Comment (bobbilee19) [reply docs/SSI/ssi-console.md]: not here anymore

**Comment by bobbilee19** [comment]
@kirahsapong @angiejones All of your suggestions / changes are in.

**Review by angiejones — APPROVED** [review]

**Comment by bobbilee19** [comment]
@angiejones Review once more, I fixed the VC blog post and tested all steps so it's good to go.

**Comment by bobbilee19** [comment]
The styling being off here i mentioned to nick and kirah last week, im guessing will be fixed soon.

Screenshot 2023-05-29 at 12 45 25 PM

**Comment by angiejones** [comment]
nice work @bobbilee19. you can merge
